### PR TITLE
handle ++/-- of emojis

### PR DESCRIPTION
**Pull request by samuelcouch**
Don't escape a trailing `:` if you are trying to ++/-- an emoji. Also some general cleanup for style consistency.

Currently: 
input -> :8ball: ++ 
output -> :8ball has 1 point

With fix:
input :8ball: ++ 
output: :8ball: has 1 point

### CODE REVIEW

File: src/plusplus.coffee [hunk 1]
```diff
@@ -51,10 +51,15 @@ module.exports = (robot) ->
...
+
+    if name
+      if name.charAt(0) == ":"
+        name = (name.replace /(^\s*@)|([,\s]*$)/g, '').trim().toLowerCase()
+      else
+        name = (name.replace /(^\s*@)|([,:\s]*$)/g, '').trim().toLowerCase()
 
     # check whether a name was specified. use MRU if not
-    unless name? && name != ''
+    unless name? && name != ""
```
Comment (ajacksified) [inline src/plusplus.coffee]: Why were the single quotes replaced here, and below?

Comment (samuelcouch) [inline src/plusplus.coffee]: Only for the aesthetic. Noticed that double quotes were used in most other places -- although in doing a more thorough check I see it's more of a 50/50 mix. I can swap those back.

**Comment by samuelcouch** [comment]
Just noticed the note about submitting changes to [the other repository](https://github.com/hubot-scripts/hubot-plusplus). Should I move my PR there?

**Comment by ajacksified** [comment]
Published as 1.1.6. Thanks!

**Comment by samuelcouch** [comment]
Sweet! :+1:

**Comment by toshimaru** [comment]
Cool. samuelcouch++

**Pull request merged**